content-service/src/main/resources/locale/portlet/news/News_fr.properties

```diff
@@ -96,7 +96,7 @@ news.unarchive.confirm=Cet article ne sera plus archiv\u00E9 et il sera de nouve
 news.unarchive.success=Cet article est de nouveau visible
 news.unarchive.error=Une erreur est apparue lorsque vous avez essay\u00E9 de rendre \u00E0 nouveau visible l'article. Essayez encore et si l'erreur persiste, veuillez contacter votre administrateur.
 
-news.notification.description={0} a post\u00E9 un article dans l'espace {2}:
+news.notification.description={0} a post\u00E9 un article dans l'espace {2}: "{1}"
```

Comment:
IMO the "en" modification is sufficient since the "fr" modification will be overridden by crowdin

Reply:
done